src/main/dynamo/Beyonce.ts

```diff
@@ -143,15 +144,15 @@ export class Beyonce {
   async batchGet<T extends PartitionAndSortKey<TaggedModel>>(params: {
     keys: T[]
     consistentRead?: boolean
-  }): Promise<GroupedModels<ExtractKeyType<T>>> {
+  }): Promise<{ items: GroupedModels<ExtractKeyType<T>>; unprocessedKeys: T[] }> {
```

Comment:
So now we return any unprocessed keys. If the requests where chunked into N concurrent requests we aggregate all the unprocessed keys.